modules/KalturaSupport/resources/webcast/userProfile.js

```diff
@@ -58,16 +58,19 @@
             // If our user ID is the same as the configured anonymousUserId we need to generate one, or get it from localStorage (if exists)
             if (!pluginGetConfig("userRole") || pluginGetConfig("userRole") === "anonymousRole"){
 
-                //if localStorage is available, get & store the user id from it;
-                if(window.localStorage) {
-                    if (!localStorage.kAnonymousUserId) {
-                        localStorage.kAnonymousUserId = _this.generateUserId(pluginGetConfig("userId"));
```

Comment:
need _this.generateUserId(pluginGetConfig("userId")) instead of _this.generateUserId()